src/web/routes/application/common/state-machine/state-machine.test.js

```diff
@@ -4,7 +4,11 @@ const { CHECK_URL, TERMS_AND_CONDITIONS_URL, CONFIRM_URL } = require('../../comm
 
 const { GET_NEXT_PATH, INVALIDATE_REVIEW } = actions
 
-const steps = [{ path: '/first', next: () => '/second' }, { path: '/second' }]
+const steps = [
+  { path: '/first', next: () => '/second' },
+  { path: '/second', next: () => '/third' },
+  { path: '/third', isNavigable: () => false, next: () => '/fourth' }
```

Comment:
Being picky here, but can we have the fourth entry on it's own line.

Reply:
That's not a fourth entry, its the `next` for the third entry.

Reply:
There is no fourth entry (deliberately). The '/fourth' path is actually the next property of the third step.